packages/terra-functional-testing/bin/test-runner.js

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env node
 
-require('../lib/cli').run();
+require('../src/cli').run();
```

Comment:
Is this going to switch back to lib in the next PR? I think we'll run into issues using /src/ when this is consumed as a released package

Reply:
Yeah, I will update when we pull babel in.